index.html

```diff
@@ -1,187 +1,284 @@
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card4">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card5">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card6">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-            </div>   
-        </section>
+          </div>
+        </div>
+      </section>
+
+      <section class="form-contact">
+        <h3 class="text-contact">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+        <form
+          id="template-form"
+          action="https://formspree.io/f/template-form"
+          method="post"
```

Comment:
- [ ] Kindly as asked by the previous reviewer, First you have to create an account on [Formspree service](https://formspree.io/html) after that add a new form like in this pic
![image](https://user-images.githubusercontent.com/35535130/159598607-d4a0045f-e082-46ef-a6e3-b80bc8599aaf.png)

After that open the integration tab and you will find your link with the id. Take it cope and replace it with this line `https://formspree.io/f/template-form`.

![image](https://user-images.githubusercontent.com/35535130/159598789-8bbac3c8-2a8a-4909-80aa-e5af88aa9cd2.png)

Try to submit first before you request a review again the page after submitting should say `Thank you` work, not `Form not found`

Reply:
@keroloussamy ooooh ok sorry about that, thank you so much! now just work fine

Reply:
@rega1237 Submit again if it's working